game_runner.rb

```diff
@@ -1,3 +1,5 @@
 require_relative "lib/game.rb"
+require_relative "lib/interface.rb"
 
-Game.run($stdout, $stdin)
+interface = Interface.new(std_out: $stdout, std_in: $stdin)
```

Comment:
I'm concerned with making this file more complex, because it has no test coverage. I think Ideally it only knows about Game, who then is responsible for building out the Interface and distributing it.

Reply:
That makes sense to me. I like thinking about it as a new thing this file has to know about a game (that it needs an interface). And perhaps a new local variable is another clue.